api/ContactAddress.json

```diff
@@ -0,0 +1,53 @@
+{
+  "api": {
+    "ContactAddress": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": "84"
+          },
+          "edge": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "13",
```

Comment:
I've confirmed there was something there in Firefox back then, but it was part of a totally different API, removed in https://github.com/mozilla/gecko-dev/commit/80af1b3196490d7e2b1e6fae66329a6462623e75, although then `ContactAddress` was a dictionary so I don't think that's the right commit for a removal version.

Since this is something totally different and a very long time ago, I'm just going to set the data for Firefox to false. If our tooling keeps suggesting adding it, we can think about a workaround to suppress it.